src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SetComplexityLevelRequestDTO.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import java.util.UUID
+
+data class SetComplexityLevelRequestDTO(
```

Comment:
Can perhaps combine ReferralComplexityLevel and SetComplexityLevelRequestDTO into a single DTO if making the assumption that DraftReferralDTO is the REST resource that is being modified.

Reply:
the dto would end up with both fields being nullable though i think? how would you structure it?

Reply:
Oh I'm not sure what you mean...

It's a really minor point...

I was just thinking you could rename ReferralComplexityLevel to ReferralComplexityLevelDTO, and use this as the REST resource that is being PATCHed rather than a separate DTO object.

Reply:
Sorry, you're right, as it's a list rather than a direct object.